gstreamer-plugin/gstsvthevcenc.c

```diff
@@ -904,7 +904,7 @@ gst_svthevcenc_encode (GstSvtHevcEnc * svthevcenc, GstVideoCodecFrame * frame)
 
   if (GST_VIDEO_CODEC_FRAME_IS_FORCE_KEYFRAME (frame)) {
     input_buffer->sliceType =
-        (svthevcenc->svt_config->intraRefreshType == 2) ?
+        (svthevcenc->svt_config->intraRefreshType >= 0) ?
```

Comment:
It seems that we can't replace 0 with IDR_REFRESH here, because EbDefinitions.h is invisible to gstsvthevcenc.c, correct?

Reply:
Yes, not in this scope.